### ISSUE TITLE
Disable Touchpad while Typing

### ISSUE THREAD

**Issue by funnylookinhat** [solus-project/repository#42]
Or provide the ability to set a delay in Mouse & Touchpad settings.

**Comment by howdystranger** [comment]
Yeah, this is a really critical feature I think. So annoying to constantly accidentally brush past the touchpad and start typing in a completely different place!
